.gitlab/scripts.yml

```
@@ -43,6 +43,7 @@
         -DGINKGO_BUILD_HIP=${BUILD_HIP}
         -DGINKGO_BUILD_MPI=${BUILD_MPI} ${MPI_STR}
         -DGINKGO_BUILD_HWLOC=${BUILD_HWLOC}
+        -DGINKGO_BUILD_PAPI_SDE=${BUILD_PAPI_SDE}
```

is this flag outdated?
```suggestion
        -DGINKGO_WITH_PAPI_SDE=${BUILD_PAPI_SDE}
```

---

Thanks! That allowed me to figure out my project file finder was somehow ignoring hidden files...

---

I moved everything back to `BUILD` and `HAVE` for consistency, since we need the two.